ref/general/os-hardening.adoc

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description:
+:seo-title: Operating system security hardening
+:seo-description:
+:page-layout: general-reference
+:page-type: general
+= Operating system security hardening
+
+Operating system (OS) intrusions occur when unauthorized users can log in to the OS and gain access to sensitive resources.
+Before you run a container in production, harden the container image to remove any attack vectors that might allow exploitation of your server's configuration or install.
+
+== Open Liberty Docker images
+A container that runs in production should have only the tools and packages that you need to run your application. You can find Open Liberty Docker images and information about them in link:https://hub.docker.com/_/open-liberty[Docker Hub].
```

Comment:
Change,
"A container that runs in production should have only the tools and packages that you need to run your application."
to,
"A container that runs in production **must** have only the tools and packages that you need to run your application."

"You can find Open Liberty Docker images and information about them in link:https://hub.docker.com/_/open-liberty[Docker Hub]." can be rephrased as "You can find Open Liberty Docker images and related information in link:https://hub.docker.com/_/open-liberty[Docker Hub]."